package.json

```diff
@@ -2,7 +2,12 @@
   "name": "",
   "version": "1.0.0",
   "scripts": {
-    "test": "elm make --docs=docs.json && elm-format src/ tests/ --validate && elm-test && elm-review && npm run package-tests -s",
+    "test": "npm-run-all -l -s test:*",
+    "test:make": "elm make --docs=docs.json",
+    "test:format": "elm-format src/ tests/ --validate",
+    "test:run": "elm-test",
+    "test:review": "elm-review",
+    "test:package": "npm run package-tests -s",
```

Comment:
This is just running `package-tests`, you might as well copy the contents of that script here and remove `package-tests` which we will have no need for anymore.

Reply:
OK